CoursesManager.UI/Database/Procedures/Registrations/spRegistrations_Add.sql

```diff
@@ -0,0 +1,16 @@
+﻿DELIMITER $$
+CREATE DEFINER=`courses_manager`@`%` PROCEDURE `spRegistrations_Add`(
+    IN p_course_id INT,
+    IN p_student_id INT,
+    IN p_registration_date DATE,
+    IN p_payment_status TINYINT,
+    IN p_is_active TINYINT,
+    IN p_is_achieved TINYINT,
+    IN p_created_at DATETIME,
+    IN p_updated_at DATETIME
```

Comment:
Die kun je er nog uithalen

Reply:
Eruit gehaald en geupdate in de database